specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/2017-06-01-preview/containerregistry.json

```diff
@@ -477,8 +477,8 @@
         "tags": [
           "Replications"
         ],
-        "description": "Creates or updates a replication for a container registry with the specified parameters.",
-        "operationId": "Replications_CreateOrUpdate",
+        "description": "Creates a replication for a container registry with the specified parameters.",
+        "operationId": "Replications_Create",
```

Comment:
This would be a breaking change but this resource type is still under private preview.